packages/server-wallet/src/wallet/multi-threaded-wallet/worker.ts

```diff
@@ -13,12 +13,13 @@ import {isStateChannelWorkerData} from './worker-data';
 const walletConfig: ServerWalletConfig = {
   ...(workerData as ServerWalletConfig),
```

Comment:
Can we get away without this type assertion?

Reply:
I think this isn't easy to do, as `workerData` is the data that's passed to the thread, so typescript can't check it, as it has no control over how the thread is called. The alternative would be to write a validation function that checks the format of the `workerData`, throwing an error if it's the wrong format and type asserting otherwise. I think that would only accomplish failing sooner though, and probably isn't worth the effort at this stage.